webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/services/impl/SystemVariableSpecServiceImpl.scala

```diff
@@ -142,6 +142,10 @@ class SystemVariableSpecServiceImpl extends SystemVariableSpecService {
                        "MANAGED_NODES_KEY" , "CFEngine KEY (modulus with transformation) of nodes managed by the policy server"
                                            , constraint = Constraint(mayBeEmpty=true)
                         )
+    , SystemVariableSpec(
+                       "POLICY_SERVER_KEY" , "CFEngine KEY (modulus with transformation) of the policy server"
```

Comment:
it is not the key, but the hash
Could you rename it to "Agent key hash of the policy server"

Reply:
it's to be consistant with the `MANAGED_NODES_KEY`, which are the same thing (and should be name otherwise, but are not)

Reply:
```suggestion
                       "POLICY_SERVER_KEY" , "Hash of CFEngine KEY of the policy server"
```